### DISCUSSION TITLE
How do I make two blogs on my website?

### DISCUSSION THREAD

**Discussion by BogusCurry** [datenstrom/community#752] [Ask a question] [answered]
Hello all ;D

I would like to put several blogs on the website, which contain different topics. How to do this ?

Greetings

Bogus | magicplace.de

**Comment by GiovanniSalmeri** [comment, marked as answer]
Hello! I think that the answer to your question is here: https://github.com/datenstrom/yellow/discussions/108#discussioncomment-244180

**Reply by BogusCurry** [reply]
Hi ;D 
Thanks for your answer ;D I will test it ;D
